src/protocols/aeternity/libs/AeAccountHdWallet.ts

```diff
@@ -92,6 +93,36 @@ export class AeAccountHdWallet extends AccountBase {
     );
   }
 
+  override async signJWT(
+    message: object,
+    options: Parameters<AccountBase['signJWT']>[1] & InternalOptions,
+  ): Promise<`${string}.${string}`> {
```

Comment:
Maybe we could name the `${string}.${string}` type and save it somewhere in `types/index.tx` file so this code would be easier to read?